Userland/Libraries/LibJS/Console.cpp

```diff
@@ -76,10 +76,14 @@ ThrowCompletionOr<Value> Console::warn()
     return js_undefined();
 }
 
-Value Console::clear()
+// 1.1.2. clear(), https://console.spec.whatwg.org/#clear
+ThrowCompletionOr<Value> Console::clear()
 {
+    // 1. TODO: Empty the appropriate group stack.
```

Comment:
This TODO looks like it is from the spec. Maybe use FIXME or add a comment below the spec text about why this cannot be implemented yet (or just that it isn't yet).

Reply:
Actually I see you implement this later on in which case I think you can just leave it.